util/containerutil/alias_test.go

```diff
@@ -0,0 +1,19 @@
+package containerutil_test
+
+import (
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	NoError  = require.NoError
```

Comment:
Why this style over `. "github.com/stretchr/testify/require"`, out of curiosity?

Reply:
it makes it possible to `grep` for the function, otherwise grepping for `Nil` won't match the `. "github.com/stretchr/testify/require"`.

We have a few older tests that still use the `. "github.com/stretchr/testify/require"` includes, which haven't been switched over.

Reply:
Not sure what you mean. Wouldn't `Nil` still be present in the test file?

Reply:
I see. Fair enough.